src/matching/Matching_generic.mli

```diff
@@ -212,6 +212,7 @@ val m_string_ellipsis_or_metavar_or_default :
   ?m_string_for_default:string matcher -> string AST_generic.wrap matcher
 
 val m_ellipsis_or_metavar_or_string : string AST_generic.wrap matcher
+val m_doc_string_metavar_or_string : string AST_generic.wrap matcher
```

Comment:
Maybe it's better to put this in `Pattern_vs_code`? 🤔 I would probably just call it `m_func_doc_comment` or sth. I guess `m_ellipsis_or_metavar_or_string` has that kind of name and placed here because it was meant to be more "generic" (although I'm not sure).

Reply:
Fair enough. I've moved the functions.